utils/generate-docs/Schema/Schema.ts

```diff
@@ -69,6 +69,21 @@ export default class Schema {
     return schemaNode;
   };
 
+  findSchemaNodeBySchemaRelativeId = (partialId: string) => {
+    const parentType = partialId.split("/")[0];
+    const candidates = this.filterSchemaNodesByParentType(parentType);
+    if (partialId === "enums/ObjectType.schema.json") {
+      console.log(parentType);
+      console.log(candidates.map((c) => c.id()).join("\n"));
+    }
+    const schemaNode = candidates.find((node) => node.id().endsWith(partialId));
```

Comment:
@pjohnmeyer, just wanted to document here for myself and others if they ever wonder the same thing - in our validator, AJV ensures that you can't have duplicate IDs in files with different paths. That said, it's possible to (erroneously) have two files with the same partialIds at the end (this could happen if someone doesn't know what they're doing or perhaps we have old and new schemas being mixed after the $ids get changed for a a release). Do we need an explicit check to ensure this is not the case? I'm going to go ahead and merge this either way though.

Reply:
I don't think so, but I think your question is a sign that this functionality needs a better name.  (although I see I left some `console.logs` in there, oops)

The `partialId` here has to start at the root of the schema, e.g. it has to be `files/...` or `enums/...` or the like. LL73-74 ensures this by first filtering all nodes by "parent type".